d2ir/d2ir.go

```diff
@@ -221,6 +221,17 @@ func (m *Map) Root() bool {
 	return f.Root()
 }
 
+func (m *Map) AbsID() string {
```

Comment:
See global `IDA()` function. Copy same code of global IDA and remove `|| NodeBoardKind(f) != ""` on the 4th line and name it `AbsIDA()` and make it a global too.

Reply:
Or actually rename the global IDA to `BoardIDA()` and make this new one just `IDA()`

Reply:
And then delete this of course and use the global.

Reply:
ah. missed that function